RecoTracker/LSTCore/interface/EndcapGeometry.h

```diff
@@ -27,7 +27,7 @@ namespace lst {
 
     void load(std::string const&);
     void fillGeoMapArraysExplicit();
-    float getdxdy_slope(unsigned int detid) const;
+    float getdxdy_slope(const unsigned int detid) const;
```

Comment:
const for args passed by value is usually not added; was there a comment?

Reply:
Not sure if there was an explicit comment about arguments by value, should I revert it?

Reply:
IIRC `const` for passing by value has no impact on the interface side signature of the methods.
It only becomes useful for long-scope implementations.

There was no separate commit. So, I'm not sure if reverting will be simple; it should be acceptable (just a bit too verbose where it's not really necessary)

Reply:
If I am looking into removing all of the non-const arguments by reference when it's unnecessary (see question above), then this is not more work.